pkg/nuclide/ui/tree/lib/TreeRootComponent.js

```diff
@@ -178,6 +190,35 @@ var TreeRootComponent = React.createClass({
     this.setState({expandedKeys});
   },
 
+  async _toggleChildNodesExpanded(parentNode: LazyTreeNode, forceExpanded: ?boolean): Promise {
```

Comment:
Not sure if I fully understands what you mean here. 
Do you want me to remove the `_toggleChildNodesExpanded` method and change `expandNodeKey` and `collapseNodeKey`to take a argument (boolean) which will expand/collapse the child nodes?